cluster-autoscaler/core/scaleup/orchestrator/orchestrator.go

```diff
@@ -196,17 +196,7 @@ func (o *ScaleUpOrchestrator) ScaleUp(
 	}
 
 	// Apply upper limits for resources in the cluster.
```

Comment:
Similar remark to the other one - I don't think there's value in keeping a comment that essentially repeats what the function name says already.

Reply:
Totally agree, changed. I just wanted to align to other comments in this function :)